src/common/Button/button.scss

```
@@ -0,0 +1,17 @@
+.button {
+    width: 150px;
+    height: 40px;
+    margin-right: 2%;
+
+    font-family: 'Open Sans';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 34px;
+    
+    background: #007298;
+    border-radius: 5px;
+    border: 0px;
```

Unit of measure 'px' is redundant
```suggestion
    border: 0;
```

---

refactored